server/notebooks/local_admin_notebook.ipynb

```diff
@@ -332,44 +326,38 @@
         },
         {
             "cell_type": "code",
-            "execution_count": 7,
+            "execution_count": 6,
             "id": "9b7a7fae",
             "metadata": {},
             "outputs": [
                 {
-                    "data": {
-                        "text/plain": [
-                            "[]"
-                        ]
-                    },
-                    "execution_count": 7,
-                    "metadata": {},
-                    "output_type": "execute_result"
+                    "name": "stdout",
+                    "output_type": "stream",
+                    "text": [
+                        "[]\n"
+                    ]
                 }
             ],
             "source": [
-                "run(\"python mongodb_admin.py show_collection --collection datasets\", to_dict = True)"
+                "run(\"python mongodb_admin.py show_collection --collection datasets\")"
```

Comment:
i can add the '_cli' afterwards

Reply:
(due to other pylint PR)

Reply:
i can adapt, i think the easier is to first merge your PR, then i will manage the potential conflict and the rename of the file.